app/models/bus.rb

```diff
@@ -1,3 +1,4 @@
+require 'sequel'
```

Comment:
how in the world did this file ever run before without this import? Removing this import doesn't seem to have any effect which scares me (I'm ensuring this file gets hit). Why'd you add it?

Reply:
It works because `sequel` is already included and in-scope when models are `require`d. check `server.rb`. I included it for intellisense.